datadog_checks_dev/datadog_checks/dev/tooling/e2e/docker.py

```diff
@@ -214,7 +218,7 @@ def start_agent(self):
                 'DD_EXPVAR_PORT=0',
                 # Run API on a random port
                 '-e',
-                'DD_CMD_PORT=0',
+                'DD_CMD_PORT={}'.format(free_port),
```

Comment:
Awesome!! Shall we print that port to stdout?

Reply:
AFAIK it's only used internally by the agent, I don't think it's worth it except if you have use case in mind?

Reply:
My bad. I though it was necessary to pass the `DD_CMD_PORT` when running `agent status`, but looks like it's already there after `docker exec` inside the container.